Google_Compute_Instance.yml

```diff
@@ -0,0 +1,888 @@
+description: This integration lets you view, create, delete, list, start and stop
+  Google Compute Instances
+detaileddescription: "Steps to run the Google Compute Instance integration\n*Getting
+  authorization credentials from Google Compute Console*\n1. Go to ‘API & Services’
+  > ‘Credentials’ from Google Compute Console left menu bar\n2. On ‘Credentials’ page,
+  from ‘Create Credentials’ dropdown select ‘Service account key’\n3. On the new page,
+  from ‘Service account’ dropdown select ‘Compute Engine default service account’
+  and select ‘JSON’ from ‘Key Type’. This will download a JSON file. \n\n*Adding instance
+  and authorizing it on Demisto*\n1. Go to Settings\n2. Search for “Google Compute
+  Instance”\n3. Click on “Add instance”, this should slide in a pop up\n4. From the
+  downloaded JSON file, enter details asked on the pop up as it is except the private_key\n5.
+  Private_key is a long string with multiple lines. To easily extract it from JSON,
+  you can follow the following steps -\n- Copy the Private_key content\n- Run a python
+  print command on your choice of python command line. This should give you the key
+  in the exact format that it is supposed to be used.\n- Use the whole content including
+  - \n\t-----BEGIN PRIVATE KEY-----\n<key>\n-----END PRIVATE KEY-----\n6. Test the
+  integration and click ‘Done’ if success\n7. Start using the integration from the
+  War Room"
+configuration:
+- display: project_id
+  name: project_id
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: private_key_id
+  name: private_key_id
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: private_key
+  name: private_key
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: client_email
+  name: client_email
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: client_id
+  name: client_id
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: client_x509_cert_url
+  name: client_x509_cert_url
+  defaultvalue: ""
+  type: 12
+  required: true
+script:
```

Comment:
Use our integration docstring template, for example `"""Imports"""` is missing here